README.md

```diff
@@ -1,4 +1,758 @@
-# eventstreams-go-sdk
-Event Streams SDK allows you to integrate your application with Event Streams RESTful APIs.
+[![Build Status](https://travis-ci.com/IBM/eventstreams-go-sdk.svg?token=eW5FVD71iyte6tTby8gr&branch=main)](https://travis.ibm.com/IBM/adminrest-go-sdk.svg?token=eW5FVD71iyte6tTby8gr&branch=main)
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+# IBM Cloud Eventstreams Go SDK Version 0.0.1
 
-Coming soon!
+## Introduction
+
+IBM Event Streams for IBM Cloud™ is a high-throughput message bus built with Apache Kafka. 
+It is optimized for event ingestion into IBM Cloud and event stream distribution between your services and applications.
+
+Event Streams provides a REST API to help connect your existing systems to your Event Streams Kafka cluster. 
+Using the API, you can integrate Event Streams with any system that supports RESTful APIs.
+
+Documentation [IBM Cloud Eventstreams Service APIs](https://cloud.ibm.com/apidocs/event-streams).
```

Comment:
That was because some removed the tag from our public api docs. So not broken by us and fixed now,


Carolyn Carpenter  1 hour ago
@Derek Howlett It looks like the apidocs_enabled tag was removed from the offering in the global catalog. If the tag isn't there, the docs don't display. I reenabled the tag (so your docs are showing), but check with your offering manager to see how the offering was updated so that it doesn't get removed again in the future. (edited)